pygsti/report/reportables.py

```diff
@@ -537,8 +537,13 @@ def evaluate_nearby(self, nearby_model):
             JAstd = self.d * _tools.fast_jamiolkowski_iso_std(
                 nearby_model.sim.product(self.circuit), mxBasis)
             JBstd = self.d * _tools.fast_jamiolkowski_iso_std(self.B, mxBasis)
-            Jt = (JBstd - JAstd).T
-            return 0.5 * _np.trace(_np.dot(Jt.real, self.W.real) + _np.dot(Jt.imag, self.W.imag))
+            J = JBstd - JAstd
+            # Old code. Keep for now, to make it easier to see correctness of new code.
+            #   Jt = J.T
+            #   val = 0.5 * _np.trace(_np.dot(Jt.real, self.W.real) + _np.dot(Jt.imag, self.W.imag))
+            # New code.
+            val = 0.5 * (_np.vdot(J.real, self.W.real) + _np.vdot(J.imag, self.W.imag))
+            return val
```

Comment:
If people are okay with the new implementation then I'll remove the comments.

Reply:
Taking Corey's approval as an OK for the new implementation; removing these comments and marking as resolved.